gradle/verification-metadata.xml

```diff
@@ -2831,56 +2841,111 @@
+         <artifact name="tika-parser-miscoffice-module-2.6.0.jar">
+            <sha256 value="901c8d77a4b054ae95203325883649f9acff2411529770554d9f64a4b61d83eb" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.tika" name="tika-parser-pdf-module" version="2.4.0">
          <artifact name="tika-parser-pdf-module-2.4.0.jar">
             <sha256 value="46b1a66b14c802b62227456d41027766835772a142d8974923b0f16dd2a7f000" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.tika" name="tika-parser-pdf-module" version="2.6.0">
+         <artifact name="tika-parser-pdf-module-2.6.0.jar">
+            <sha256 value="7900c08e952407b1754c1e07232d8d72a01e51e3ba602fcbe9e6310b2a476f3e" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.tika" name="tika-parser-text-module" version="2.4.0">
          <artifact name="tika-parser-text-module-2.4.0.jar">
             <sha256 value="bea22bb1ea5b6a3a18d7b5a3bbf0689f2becf7568799f0cd2965cb39348959e2" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.tika" name="tika-parser-text-module" version="2.6.0">
+         <artifact name="tika-parser-text-module-2.6.0.jar">
+            <sha256 value="15b7e20c51ed14d1dac25ba9b7d1179603a36f93204e7f445c0ab98e41b14d83" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.tika" name="tika-parser-xml-module" version="2.4.0">
          <artifact name="tika-parser-xml-module-2.4.0.jar">
             <sha256 value="53e14c28b6b0b1d76b8b901cc3c8f4f04f12e5e5adfec4aee73a53fa82b83980" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.tika" name="tika-parser-xml-module" version="2.6.0">
+         <artifact name="tika-parser-xml-module-2.6.0.jar">
+            <sha256 value="124d3ef1a35fc368bc3a49d616821bfe997e95e50f1c991b7d7076b7e9546462" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.tika" name="tika-parser-xmp-commons" version="2.4.0">
          <artifact name="tika-parser-xmp-commons-2.4.0.jar">
             <sha256 value="361c615f0c5322ba786d2b06274f9946f5ec8de7784edf4fa26375f755299a45" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.tika" name="tika-parser-xmp-commons" version="2.6.0">
+         <artifact name="tika-parser-xmp-commons-2.6.0.jar">
+            <sha256 value="b96bb7d066ced0a88cf3223363642dcba3d581592d5b0770ed197e7520f5bbf0" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.tika" name="tika-parser-zip-commons" version="2.4.0">
          <artifact name="tika-parser-zip-commons-2.4.0.jar">
```

Comment:
Can you remove the elder versions from this file (assuming they are no longer needed)